services/requester/requester.go

```diff
@@ -125,13 +132,38 @@ func NewEVM(
 		)
 	}
 
+	head := &types.Header{
+		Number:   big.NewInt(20_182_324),
+		Time:     uint64(time.Now().Unix()),
+		GasLimit: 30_000_000,
+	}
+	emulatorConfig := emulator.NewConfig(
+		emulator.WithChainID(config.EVMNetworkID),
+		emulator.WithBlockNumber(head.Number),
+		emulator.WithBlockTime(head.Time),
+	)
+	evmSigner := emulator.GetSigner(emulatorConfig)
+	validationOptions := &txpool.ValidationOptions{
+		Config: emulatorConfig.ChainConfig,
+		Accept: 0 |
+			1<<types.LegacyTxType |
+			1<<types.AccessListTxType |
+			1<<types.DynamicFeeTxType |
+			1<<types.BlobTxType,
+		MaxSize: models.TxMaxSize,
+		MinTip:  new(big.Int),
+	}
```

Comment:
why do we want to have these config here in the requester? I think it would be better if we just place it in the `models/transaction` with the ValidateTransaction function on the top as a variable

Reply:
Because we need access to `config.EVMNetworkID`. This means that even if we pass the config as a parameter, we will have to construct all these objects every time we call the `ValidateTransaction` function.